esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -6450,6 +6478,9 @@ bool CWsDfuEx::onDFUFileCreateV2(IEspContext &context, IEspDFUFileCreateV2Reques
 
 bool CWsDfuEx::onDFUFilePublish(IEspContext &context, IEspDFUFilePublishRequest &req, IEspDFUFilePublishResponse &resp)
 {
+#ifdef _CONTAINERIZED
+    UNIMPLEMENTED_X("CONTAINERIZED(DFUFilePublish)"); //See JIRA issue HPCC-25172
```

Comment:
Why is this disabled? - The jira reference is useful (assuming it is correct), but the comments could do with detailing what function is needed.
Do these exceptions help?  The code should already throw an exception if it accesses something it shouldn't in the containerized builds.  It gives a list of places that should be tested before shipping 8.2, but it also means that if the code that they call is fixed you need to then remove all exceptions like this.  You could end up with code failing just because the exception hasn't been removed.
@richardkchapman what do you think?